README.md

```diff
@@ -22,6 +22,12 @@
 
 See [DESIGN.md](DESIGN.md) for an overview of features and future goals.
 
+### Breaking Changes
+
+Pion WebRTC v2.0.0 is coming soon! HEAD will be unstable for the next few weeks as we prepare for the new release. See the [release notes](https://github.com/pions/webrtc/wiki/v2.0.0-Release-Notes) to learn about the new features breaking changes.
+
+Use the tag [v1.2.0](https://github.com/pions/webrtc/tree/v1.2.0) if you'd like to continue using the 1.0 api.
```

Comment:
I would add: "... continue using the 1.0 API, but please note: once v2.0.0 is released, the v1.x line will be deprecated and unmaintained."